plugins/woocommerce-admin/client/marketplace/components/constants.ts

```diff
@@ -17,7 +17,5 @@ export const MARKETPLACE_COLLABORATION_PATH =
 	MARKETPLACE_HOST +
 	'/document/managing-woocommerce-com-subscriptions/#transfer-a-woocommerce-com-subscription';
 export const WP_ADMIN_PLUGIN_LIST_URL = ADMIN_URL + '/plugins.php';
-export const WOO_CONNECT_PLUGIN_INSTALL_URL =
-	'https://woo.com/woocom-plugin/install/';
 export const WOO_CONNECT_PLUGIN_DOWNLOAD_URL =
-	'https://woo.com/woocom-plugin/download/';
+	'https://woo.com/products/woo-update-manager/';
```

Comment:
This URL has the format of a public listing on Woo.com (/products/<product-slug>), rather than a direct download link. How about changing it smth like `products/woo-update-manager/download`?

Reply:
Fixed with https://github.com/woocommerce/woocommerce/pull/44279/commits/ecf4be033b52276ca3b15a9906b0b409aa7d1800